BAC0/core/devices/Device.py

```diff
@@ -250,7 +249,7 @@ def simulated_points(self):
         :rtype: BAC0.core.devices.Points.Point
         """
         for each in self.points:
-            if each.properties.simulated:
+            if each.properties.simulated[0]:

```

Comment:
it seems like the log was incorrect here - `bool( (False, 10) ) == True`  in Python, so I believe we actually want to be checking the first element of the tuple